docs/execution/execute.rst

```diff
@@ -33,18 +33,18 @@ You can pass context to a query via ``context``.
 
 
 Variables
-_______
+_________
 
 You can pass variables to a query via ``variables``.
 
 
 .. code:: python
 
     class Query(graphene.ObjectType):
-        user = graphene.Field(User)
+        user = graphene.Field(User, id=graphene.ID(required=True))
 
-        def resolve_user(self, info):
-            return info.context.get('user')
+        def resolve_user(_, info, id):
```

Comment:
suggest keeping `self` here. It's python convention. Plus we have it in the example above this for `resolve_name`.

Reply:
As per other discussions it looks like we're going to standardise on `root` for top level resolvers so I'll change this.